tensorflow/compiler/mlir/xla/transforms/xla_legalize_to_linalg.cc

```diff
@@ -192,6 +192,107 @@ class ScalarPointwiseToStandardConverter : public OpConversionPattern<LhloOp> {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// xla_lhlo.convolution conversion pattern.
+//===----------------------------------------------------------------------===//
+
+/// Converts xla_lhlo.convolution operation to a linalg.conv op.
+struct ConvToLinalgConverter : public OpConversionPattern<xla_lhlo::ConvOp> {
+ public:
+  using OpConversionPattern<xla_lhlo::ConvOp>::OpConversionPattern;
+
+  //  This code has been adapted from IREE's
+  //  (https://github.com/google/iree/) xla_hlo -> linalg conversion.
+  LogicalResult matchAndRewrite(
+      xla_lhlo::ConvOp op, ArrayRef<Value> args,
+      ConversionPatternRewriter& rewriter) const final {
+    if (const auto& dimensionNumbers = op.dimension_numbers()) {
```

Comment:
Can you add a comment for what this `if` is doing?

Reply:
My assumption was that if this field wasn't present, there is a default ordering for all the dimensions - in line with https://www.tensorflow.org/xla/operation_semantics#conv_convolution So this is just propagating that info. Added a comment.